CHANGELOG.md

```diff
@@ -3,6 +3,15 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.2.0] - 2024-10-16
+### Changed
+- Updated all dependency versions for hive, hadoop, datanucleus, reflections, guava, tez, mockito, junit and added them in pom.xml <properties> section
+- Updated all maven plugin versions
+- Adjusted code for Java21 and removed Guava Optional from the code
+- Changed Hive property names in StandaloneHiveServerContext
```

Comment:
Is it useful for the end users to know which ones were changed?

Reply:
No...names are the same..they just added "_"